src/go/k8s/main.go

```diff
@@ -53,8 +56,9 @@ func main() {
 	var (
 		clusterDomain               string
 		metricsAddr                 string
-		enableLeaderElection        bool
 		probeAddr                   string
+		pprofAddr                   string
```

Comment:
i don't ee this use. default to 127.0.0.1 is not use in the listen adn serve below.

Reply:
Thanks, nice catch.

Reply:
It's used in the Server definition on line 117